code/game/objects/items/tools/extinguisher.dm

```diff
@@ -44,7 +44,7 @@
 /obj/item/tool/extinguisher/attack_self(mob/user as mob)
 	safety = !safety
 	icon_state = "[sprite_name][!safety]"
-	balloon_alert(user, "[safety ? "on" : "off"]")
+	balloon_alert(user, "Safety [safety ? "off" : "on"]")
```

Comment:
?? Why are you flipping the messages

Reply:
Because it's broken currently, it says "off" when you turn it on and "on" when you turn it off. It's just such a small fix I bundled it in.

Reply:
no, it says off when you turn the safety off, i.e. you can now use it.
You're making it say "safety off" when the safety is now on, so you can't use it.

Reply:
nio hes doing it right safety gets set just before this